xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/menus_content.vm

```diff
@@ -168,9 +168,11 @@
           <span class="sr-only">$services.localization.render('core.menu.toggleDropdown')</span>
         </button>
       #end
-      <ul class="dropdown-menu #if ("$!actionUrl" == '')dropdown-menu-right#end">
+      ## We use another component for the more action menu, since it has
+      ## a more complex structure with submenus.
+      <dl class='dropdown-menu dropdown-menu-right'>
```

Comment:
Why did you remove this check on the action URL?

Reply:
Long story short, I made some changes and 'reverted' them, but left a diff afterwards.
This class changes the alignement of the dropdown relative to the button, it was not critical but I didn't notice it when comparing them on: https://github.com/xwiki/xwiki-platform/pull/2190#issuecomment-1653152654
Addressed in https://github.com/xwiki/xwiki-platform/pull/2190/commits/09d9f3b1600f44cfb67bcc5e7335a57b55837889